README.md

```diff
@@ -104,7 +108,7 @@ You're a good person and decide to validate your HTML. You copy and paste the co
 The validator throws errors at you! Why? Assuming you haven't made any mistakes in your code, how could you go about accurately validating your HTML?
 
 ```
-Your answer...
+I assume because you are using a HTML validator, containing embedded ruby is why you are getting errors. I could be lazy and take the erb code and validate that way, and use rails errors to spot errors in rails code but that seems like the long way to do it. If you are asking about validating it together I will research google. If not then maybe I read this question the wrong way.
 ```
 
```

Comment:
:+1:  You're correct in your assumptions, the question is asking how you validate the html generate by rails, since ERB tags (clown hats) are not valid HTML.  Instead you can "view source" in the browser and copy the HTML there, which is rendered as a result of Rails processing and replacing the ERB tags.